sunpy/timeseries/sources/fermi_gbm.py

```diff
@@ -59,10 +59,40 @@ class GBMSummaryTimeSeries(GenericTimeSeries):
     # Class attribute used to specify the source class of the TimeSeries.
     _source = 'gbmsummary'
 
+    def plot(self, axes=None, **kwargs):
+        """
+        Plots the GBM timeseries with data from its pandas dataframe.
+
+        Parameters
+        ----------
+        axes : `matplotlib.axes.Axes`, optional
+            The axes on which to plot the TimeSeries. Defaults to current axes.
+        **kwargs : `dict`
+            Additional plot keyword arguments that are handed to `~matplotlib.axes.Axes.plot`
+            functions.
+
+        Returns
+        -------
+        `~matplotlib.axes.Axes`
+            The plot axes.
+        """
+        self._validate_data_for_plotting()
+        if axes is None:
+            axes = plt.gca()
+        data_lab = self.to_dataframe().columns.values
+        for d in data_lab:
+            axes.plot(self.to_dataframe().index, self.to_dataframe()[d], label=d, **kwargs)
+        axes.set_yscale("log")
+        axes.set_xlabel('Start time: ' + self.to_dataframe().index[0].strftime('%Y-%m-%d %H:%M:%S UT'))
+        axes.set_ylabel('Counts/s/keV')
+        axes.legend()
+        return axes
+
     @peek_show
     def peek(self, **kwargs):
```

Comment:
This is lacking a title argument, if all the other peeks have one.

Reply:
Didn't we decide to not add one?
Do I add it in xD

Reply:
I don't see why some peek methods should have it and some don't.

Reply:
Yeah, the some of the peek methods had it initially as well, I thought we wouldn't be changing how it was before, so I left it in

Reply:
I think we wanted to not break the old API but we should also try to unify them where possible.

Reply:
> > > Discussion in community meeting:
> > > Make sure we don't add or break/remove the original plot/peek api for timeseries.
> > > Move all custom plotting to plot as required.
> > > We would need to deal with plots that have multiple axes but I think we can handle that.
> > 
> > 
> > So, just removing the title arguments?
> 
> If thats all the difference is, then yes.

Okay, so add title arguments for every peek method but not plot right?

Reply:
Pretty much.

Reply:
Alright, I'll add that in.

Reply:
Thanks.